code/modules/integrated_electronics/core/prefab.dm

```diff
@@ -0,0 +1,160 @@
+/obj/item/device/integrated_electronics/prefab
+	var/debug = 0

```

Comment:
There's a better way than a debug variable in a object for this stuff. Someone else can explain.

Reply:
If it's comfortable way to debug for me, I shouldn't change it,should I?

Reply:
The debug stuff like that is usually removed before committing changes.

Reply:
Maybe I shouln't. I sure, that togglable debug will save a lot of work.

Reply:
it's fine for now, just remove when testing is complete